src/components/Toolbar.tsx

```diff
@@ -55,7 +55,7 @@ const Toolbar = ({ dark, fontSize, toolbarOverlay, scrollPrioritized, showTopCon
   const [lastScrollLeft, setLastScrollLeft] = useState<number | undefined>()
   const [leftArrowElementClassName = 'hidden', setLeftArrowElementClassName] = useState<string | undefined>()
   const [rightArrowElementClassName = 'hidden', setRightArrowElementClassName] = useState<string | undefined>()
-
+  const [pressingToolbarId, setPressingToolbarId] = useState('')
```

Comment:
Default state should be `null` which makes it more explicit. Empty string is more error prone.

```ts
useState<null | string>(null)
```